lib/sum_of_minimum_balances/main.py

```diff
@@ -1,97 +1,124 @@
 #!/usr/bin/env python3
-
+import asyncio
 import os
 from datetime import datetime, timedelta
-from typing import Optional
+from typing import Optional, Tuple
+import pandas as pd  # type: ignore
 
-from pngme.api import Client
+from pngme.api import AsyncClient
 
 
-def get_sum_of_minimum_balances(
-    api_client: Client, user_uuid: str, utc_starttime: datetime, utc_endtime: datetime
-) -> Optional[float]:
-    """Sum observed minimum balances across all depository accounts.
+async def get_sum_of_minimum_balances(
+    api_client: AsyncClient,
+    user_uuid: str,
+    utc_time: datetime,
+    cutoff_interval: timedelta = timedelta(days=90),
+) -> Tuple[Optional[float]]:
+    """Sum observed minimum balances across all depository accounts over a given time window.
 
     Args:
-        api_client: Pngme API client
-        user_uuid: the Pngme user_uuid for the mobile phone user
-        utc_starttime: the datetime for the left-hand-side of the time-window
-        utc_endtime: the datetime for the right-hand-side of the time-window
+        api_client: Pngme Async API client
+        user_uuid: Pngme mobile phone user_uuid
+        utc_time: the time at which the balances are computed
+        cutoff_interval: if balance hasn't been updated within this interval, then balance record is stale, and method returns None
```

Comment:
We should explain the definition of the argument, and then talk about edge case.